Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.py

```diff
@@ -330,6 +330,27 @@ def delete_u2f_token(token_id):
     demisto.results('Token with ID ' + token_id + ' deleted successfully')
 
 
+def set_user_status(username, status):
+    result = admin_api.get_users_by_name(username)
+    user_id = result[0]["user_id"]
+    res = admin_api.update_user(user_id, status=status)
+    demisto.results('User ' + username + 'was set to status ' + status)
+
+
+def get_duo_admin_by_name(name):
+    duo_administrators = admin_api.get_admins()
+    duo_administrator = next((admin for admin in duo_administrators if admin['name'] == name), None)
+
+    entry = get_entry_for_object(
+        'Information about admin ' + name, duo_administrator, duo_administrator,
+        {
+            'DuoAdmin.AdminDetail(val.name && val.name==obj.name)':
+                {'details': duo_administrator}
```

Comment:
Few notes:
1. You dont have a `DuoAdmin.AdminDetail.name` field in that case but ``DuoAdmin.AdminDetail.details.name``. either add it to the context as follows or fix the path.
```suggestion
            'DuoAdmin.AdminDetail(val.name && val.name==obj.name)':
                {
                'name': name,
                'details': duo_administrator
                }
```
2. What data does `duo_administrator` hold? please add the related fields to the command's outputs in the yml file. 
you can use `json_to_output` for that matter. You can see an example and explanation for this [here](https://xsoar.pan.dev/docs/integrations/context-and-outputs#integration-yml)